src/Collector/ProfileClient.php

```diff
@@ -173,6 +173,10 @@ private function collectResponseInformations(ResponseInterface $response, Stopwa
         $stack->setDuration($event->getDuration());
         $stack->setResponseCode($response->getStatusCode());
         $stack->setClientResponse($this->formatter->formatResponse($response));
+        if ($response->hasHeader('X-Debug-Token-Link')) {
```

Comment:
Should we check for both headers that we request below?

Reply:
I don't think it's necessary. The link is mandatory, but if `X-Debug-Token` is not provided all that happens is the label is shortened to `Debug Token: ` which admittedly is not ideal but would still work